dapp/src/components/evm/UnlockNftList.tsx

```diff
@@ -0,0 +1,436 @@
+"use client";
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Card,
+  CardActions,
+  CardContent,
+  CardMedia,
+  CircularProgress,
+  Stack,
+  Typography,
+} from "@mui/material";
+import TransactionButton from "../TransactionButton";
+import { useWaitForTransaction } from "wagmi";
+import { useGetLocksEVM } from "../../hooks/evm/useGetLocksEVM";
+import { LockInfoEVM, TokenEVM } from "../../utils/evm/types";
+import Grid from "@mui/material/Unstable_Grid2";
+import { hololockerConfig } from "../../utils/evm/contracts";
+import { Countdown } from "../Countdown";
+import { useInterval } from "usehooks-ts";
+import { useEffect, useState } from "react";
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+import { useGetNftsMetadataEVM } from "../../hooks/evm/useGetNftsMetadataEVM";
+import { NftTokenType } from "alchemy-sdk";
+import CollectionUnlockButton from "./CollectionUnlockButton";
+import CollectionWithdrawButton from "./CollectionWithdrawButton";
+import { useQueryClient } from "@tanstack/react-query";
+import FunctionKey from "../../utils/functionKey";
+import { areEqualTokens, formatEVMAddress } from "../../utils/evm/utils";
+import MultipleSelectionUnlockButton from "./MultipleSelectionUnlockButton";
+import MultipleSelectionWithdrawButton from "./MultipleSelectionWithdrawButton";
+import { useSnackbar } from "notistack";
+import { SnackbarMessage } from "../../utils/texts";
+import CopyableTypography from "../CopyableTypography";
+import { writeContract } from "@wagmi/core";
+
+// average block time on ETH
+const blockTime = 12n;

```

Comment:
I know we aren't support other chains automatically because of the alchemy endpoints, but I'd be still really careful about hardcoding things for Ethereum L1 like this since it will be easy to miss if we ever want to deploy this for other EVM layers (Astar, Avalanche, etc.) that have NFT projects that may want to work with us